src/renderers/common/Textures.js

```diff
@@ -136,6 +147,7 @@ class Textures extends DataMap {
 
 				const texture = textures[ i ];
 
+				texture.isTextureArray = size.depth > 1;
```

Comment:
I have investigated why some many examples fail in the CI and the changes to this file are the root cause. Cube render targets have cube textures and their size ends up with a depth value of `6`. So the the check `size.depth > 1` isn't sufficient to go into the texture array code path. 

Can we do this instead:
```js
const useArrayTexture = renderTarget.isXRRenderTarget && size.depth > 1;
```

This flag can be evaluated to create `DepthArrayTexture` and set the `isTextureArray`.

Reply:
that doesn't work unfortunately since the current render target is the one for the first pass so not the XR one.